### PR TITLE
Added a Review button. Updated UI design.

### PR DESCRIPTION
**Pull request by chc5**
Hours Spent: 2
Updates:
- Added a Review Button.
- Updated the UI design by removing and updated CSS that prevented a responsive UI.
- Updated snapshots accordingly.

### CODE REVIEW

File: client/src/User/UserHome/UserHome.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ class UserHome extends Component {
 
   render() {
     return (
-      <div style={{ padding: "50px 200px" }}>
+      <div style={{ margin: "5%" }}>
```
Comment (gerryzqx) [inline client/src/User/UserHome/UserHome.js]: Why did you change the styling to margin 5%? The original styling with padding: "50px 200px"make the layout look even throughout our app.

Comment (chc5) [reply client/src/User/UserHome/UserHome.js]: Example
If you take a look at the website currently in production and resize your window, the padding will remain fixated on padding the div by 200 pixels. Instead of having static styling, it's better to change the entire User Interface based on percentages or viewport which makes the entire website responsive. When we make our website responsive, it will also solve our issue in displaying our website on the mobile device.

**Comment by zhuoyingcai** [comment]
I tried to resize the browser and everything scaled responsively. The review button worked as expected. However, I think we have to design the UI together to make the software more user friendly.

**Comment by chc5** [comment]
Yes, I agree.

File: client/src/User/UserHome/UserHome.js (hunk 1 above)
Comment (gerryzqx) [reply client/src/User/UserHome/UserHome.js]: Oh yes, that make sense. Thanks for clarifying this up!
